index.js

```diff
@@ -0,0 +1,153 @@
+      el.parentElement.parentElement.remove();
+    }
+  }
+
+  static showAlert(message, className) {
+    const div = document.createElement('div');
+    div.className = `alert alert-${className}`;
+    div.appendChild(document.createTextNode(message));
+    const container = document.querySelector('.firstContainer');
+    // const form = document.querySelector('#book-input');
+    const navigation = document.querySelector('.navigation');
+    container.insertBefore(div, navigation);
+    setTimeout(() => document.querySelector('.alert').remove(), 3000);
+  }
+
+  static clearField() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
+}
+
+const listBook = document.getElementById('listBooks');
+const newBook = document.getElementById('newBooks');
+const contact = document.getElementById('contact');
+const navItems = [listBook, newBook, contact];
+const listSection = document.getElementById('list-section');
+const addSection = document.getElementById('add-section');
+const contactSection = document.getElementById('contact-section');
+const sections = [listSection, addSection, contactSection];
+
+const dateTime = document.querySelector('#dateTime');
+
+function saveActiveNavItemLocally(id) {
+  localStorage.setItem('activeNavItem', id);
+}
+
+document.addEventListener('DOMContentLoaded', task.currentBooks);
+document.querySelector('#book-input').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  if (title === '' || author === '') {
+    task.showAlert('fill in all the fields');
+  } else {
+    const book = new Books(title, author);
+    task.addBookList(book);
+    Store.addBook(book);
+    task.showAlert('book added');
+    task.clearField();
+  }
+});
+
+document.querySelector('#list-book').addEventListener('click', (e) => {
+  task.deleteBook(e.target);
+  Store.removeBook(e.target.parentElement.previousElementSibling.textContent);
+  task.showAlert('Book Removed');
+});
+
+function displaySection(id) {
+  sections.forEach((section) => {
+    if (section.id === id) {
+      section.classList.remove('d-none');
+    } else {
+      section.classList.add('d-none');
+    }
+  });
+}
+
+function activateNavItem(id) {
+  navItems.forEach((navItem) => {
+    if (navItem.id === id) {
+      navItem.classList.add('li-active');
+    } else {
+      navItem.classList.remove('li-active');
+    }
+  });
+}
+
+// eslint-disable-next-line no-unused-vars
+function getSectionId(navItemId) {
+  let sectionId;
+  switch (navItemId) {
+    case 'listBooks':
+      sectionId = 'list-section';
+      break;
+    case 'newBooks':
+      sectionId = 'add-section';
+      break;
+    case 'contact':
+      sectionId = 'contact-section';
+      break;
+    default:
+      sectionId = '';
+  }
+  return sectionId;
+}
+
+// window.addEventListener('load', () => {
```

Comment:
- It's better to remove all commented lines to make the project more clean

Reply:
Thank you for the review.